src/styleguide/_patterns/04-templates/00-page-layouts/01-error-page.mustache

```diff
@@ -0,0 +1,11 @@
+{{> organisms-page-header }}
+<main class="grav-o-container">
+  <section class="grav-o-section--centered">
+    <h1>{{> atoms-ascii-art }} {{title}}</h1>
+    <p>{{excerpt.short}}</p>
+    <div class="grav-o-container">
+      <a href="#the-hiring-process" class="grav-c-cta">Go to the homepage</a>
```

Comment:
I think the `href` is wrong? can we put stuff in variables?

Reply:
Whoops! I'll fix that